<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimetableEditWidget</class>
 <widget class="QWidget" name="TimetableEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="removePushButton">
         <property name="text">
          <string>Удалить запись</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="applyPushButton">
         <property name="text">
          <string>Применить изменения</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Добавление записи</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="semesterLabel">
          <property name="text">
           <string>Семестр</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="dayLabel">
          <property name="text">
           <string>День</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="timeLabel">
          <property name="text">
           <string>Время</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="timeComboBox"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="audienceLabel">
          <property name="text">
           <string>Аудитория</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="semesterSpinBox"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="groupLabel">
          <property name="text">
           <string>Группа</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="teacherLabel">
          <property name="text">
           <string>Преподаватель</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="audienceComboBox"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="dayComboBox"/>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="groupComboBox"/>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="addPushButton">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="teacherComboBox"/>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="subjectComboBox"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="subjectLabel">
          <property name="text">
           <string>Предмет</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
